### DISCUSSION TITLE
How come the limit of farmeable islands is 10?

### DISCUSSION THREAD

**Discussion by 0xDirmes** [Sau1707/ModernBot#34] [General]
Just wondering whats limiting it to more islands

**Comment by Sau1707** [comment]
I just put a random number to limit the amount of calls, this is done only for the farming without the Commander since raise a lot of suspicion if they check for a bot.

If you wanna change it, feel free to do that but pay attention

**Reply by 0xDirmes** [reply]
Oh makes sense (thanks for the quick response), so would you say that if the commander is being used, it should be fine to increase? Was also thinking on increasing the sleep time to 15 minutes or so (while maintaining the randomness) to make it a bit less effecrive

**Comment by Sau1707** [comment]
If you use the commander then the resources are claimed in a different way that it's more difficult to catch, you see this if the timer it's green (yellow if it's not using the commander) 

The Bot for the resources simulate all the action that a user will perform even if you don't see them

**Reply by 0xDirmes** [reply]
Oh I thought the limit also applied to commander, thanks so much for clarifying!  Great sccript
